templates/package/package_base.mustache

```diff
@@ -39,9 +39,27 @@
     </div>
     <div class="qpy-card-content">
         <h5 class="qpy-card-title">{{name}} <small class="text-muted">{{namespace}}</small></h5>
-        <p class="qpy-card-description">
```

Comment:
Einen div-Wrapper mit der Klasse `qpy-card-description` solltest du wahrscheinlich behalten, falls man mal (z.B. in einem Theme) das Styling der Beschreibung anpassen möchte.